synapse/replication/tcp/client.py

```diff
@@ -370,15 +370,23 @@ async def wait_for_stream_position(
         # We measure here to get in flight counts and average waiting time.
         with Measure(self._clock, "repl.wait_for_stream_position"):
             logger.info(
-                "Waiting for repl stream %r to reach %s (%s)",
+                "Waiting for repl stream %r to reach %s (%s)" "; currently at: %s",
```

Comment:
looks like black did a thing here
```suggestion
                "Waiting for repl stream %r to reach %s (%s); currently at: %s",
```